src/components/Reports/TasksTable/TasksTable.jsx

```diff
@@ -37,54 +67,42 @@ export const TasksTable = ({ WbsTasksID }) => {
     setFilters(prevState => ({ ...prevState, [filterName]: value }));
   };
 
-  const FilterOptions = ({ filterName, width }) => {
-    var filtersOptions = [...Array.from(new Set(get_tasks.map(item => item[filterName]))).sort()];
-    return (
-      <DropDownSearchBox
-        items={filtersOptions}
-        searchCallback={value => setOneFilter(filterName, value)}
-        placeholder={`Any ${filterName}`}
-        className="tasks-table-filter-item tasks-table-filter-input"
-        width={width}
-        value={filters[filterName]}
-      />
-    );
-  };
-
-  const UserOptions = ({ tasks }) => {
-    let users = [];
-    tasks.forEach(task => task.resources?.forEach(resource => users.push(resource.name)));
-
-    users = Array.from(new Set(users)).sort();
-    return (
-      <DropDownSearchBox
-        items={users}
-        placeholder={`Any user`}
-        searchCallback={value => setOneFilter('users', value)}
-        className="tasks-table-filter-item tasks-table-filter-input"
-        value={filters.users}
-      />
-    );
-  };
-
   return (
     <div>
       <div>
         <h4 className="tasks-table-header">Tasks</h4>
       </div>
       <div className="tasks-table-filters-wrapper">
         <div className="tasks-table-filters">
-          <UserOptions tasks={get_tasks} />
-          <FilterOptions filterName={'classification'} width="180px" />
-          <FilterOptions filterName={'priority'} />
-          <FilterOptions filterName={'status'} />
-
+          {/* eslint-disable-next-line camelcase */}
```

Comment:
Looks like you already got the camelcase errors fixed, please also delete line 77/80/88/95